### ISSUE TITLE
text is added to hiddent tabs in Firefox

### ISSUE THREAD

**Issue by antonkulaga** [polynote/polynote#995]
In Firefox when switching between tabs (usually to edit or copy-paste from one notebook to another) after one or two switches the editor does not write to the cell of the opened tab but writes to the cell of the hiddent tab.

**Comment by jeremyrsmith** [comment]
To clarify: the issue is that when the tab is switched, the focus doesn't move to the active cell in the selected tab automatically. Instead, the previously selected tab still has focus (even though it is no longer in the DOM, I guess?) and still receives the input events.

Can probably fix by explicitly focusing the active cell in the selected tab on tab switch.

**Comment by antonkulaga** [comment]
Yes, the focus stays in the hidden cell so whatever I type goes there instead of selected one
